docs/xls-66d-lending-protocol/index.page.tsx

```diff
@@ -0,0 +1,106 @@
+import * as React from "react";
+import {
+  LandingContainer,
+  LandingLayout,
+  FeatureHeader,
+  FeatureContent
+} from "../../components/landing";
+import { AmendmentTracker } from "../../components/AmendmentTracker";
+import { Button } from "@redocly/theme";
+import { Card } from "@redocly/theme/markdoc/components/Cards/Card";
+import { Cards } from "@redocly/theme/markdoc/components/Cards/Cards";
+
+export const frontmatter = {
+  seo: {
+    title: "XLS-66d Lending Protocol",
+    description: "An XRP Ledger DeFi primitive that enables loans from a Single Asset Vault."
+  },
+};
+
+export default function Page() {
+  const KEY_DATE_EVENTS = [
+    "XLS Spec Live",
+    "Available to Test on Devnet",
+    "Open for Voting on Mainnet", 
+    "Vote Consensus"
+  ];
+
+  const [keyDates, setKeyDates] = React.useState(
+    KEY_DATE_EVENTS.map(event => ({ date: "🔄 Loading...", event }))
+  );
+
+  const handleKeyDatesUpdate = React.useCallback((newKeyDates: any[]) => {
+    setKeyDates(newKeyDates);
+  }, []);
+
+  return (
+    <LandingLayout>
+      <LandingContainer>
+        <FeatureHeader 
+          title="XLS-66d Lending Protocol"
+          subtitle="An XRP Ledger DeFi primitive that enables loans from a Single Asset Vault."
+        />
+
+        <FeatureContent 
+          description="The Lending Protocol is an XRP Ledger DeFi primitive that enables on-chain, fixed-term, uncollateralized loans using pooled funds from a Single Asset Vault. The implementation relies on off-chain underwriting and risk management to assess the creditworthiness of borrowers, but offers configurable, peer-to-peer loans without intermediaries like banks or financial institutions."
+          keyDates={keyDates}
+        />
+
+        <AmendmentTracker 
+          amendmentId=""
+          xlsSpecDate=""
+          onKeyDatesUpdate={handleKeyDatesUpdate}
+        />
+
+        <Cards columns={2}>
+          <Card
+            title="XLS Spec"
+            to=""
```

Comment:
I guess it's still in PR, but would be good to add the link to the XLS here: https://github.com/Tapanito/XRPL-Standards/blob/732b33e3eb1e05e77500367efb97eebbdcff3a93/XLS-0066d-lending-protocol/README.md